### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by iizrailevsky**
Added doc on packaging without running integration tests, if needed

### CODE REVIEW

**Comment by abhishekjain88** [comment]
Looks good 👍 :shipit: @iizrailevsky

File: README.md [hunk 1]
```diff
@@ -364,9 +364,22 @@ Code changes can readily be verified by running the growing collection of includ
 
 Wasabi can readily be packaged as installable *rpm* or *deb* distributions and deployed at scale as follows:
 
+> Package by running integration tests 1st:
+
+```bash
+% ./bin/wasabi.sh start package
```
Comment (bargenilesh) [inline README.md]: @iizrailevsky Is 'start' needed while packaging (./bin/wasabi.sh start package) ? 
If yes then, does it needed while packing without integration test ?

Comment (iizrailevsky) [inline README.md]: @bargenilesh Yes, we discovered with @abhishekjain88 the 'package' command by default runs integration tests. So it assumes a running docker container with wasabi, started with 'start' command. 

If we're not running integration tests, then there's no need to start containers explicitly.

Comment (bargenilesh) [inline README.md]: @iizrailevsky Got it, thx... Changes looks good... :shipit:

**Pull request merged**